File: python3/iniciante/1101.py
while True:
    m, n = list(map(int, input().split()))
    if n <= 0 or m <= 0:
        break

    maior, menor = m, n
    if n > m:
        maior, menor = n, m

    sum = 0
    for i in range(menor, maior + 1):
        sum += i
        print(i, end=' ')

    print("Sum={}".format(sum))

File: c++/iniciante/1101.cpp
#include <iostream>

int main() {
    int m, n, maior, menor, sum;
    while (1) {
        std::cin >> m;
        std::cin >> n;
        if (n <= 0 || m <= 0) {
            break;
        }

        maior = m;
        menor = n;
        if (n > m) {
            maior = n;
            menor = m;
        }

        sum = 0;
        for (int i = menor; i <= maior; i++) {
            sum += i;
            std::cout << i << " ";
        }

        std::cout << "Sum=" << sum << std::endl;
    }

    return 0;
}
